src/view/com/util/LoadingScreen.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import {ActivityIndicator, View} from 'react-native'
+import {s} from 'lib/styles'
+import {CenteredView} from './Views'
+
+export function LoadingScreen() {
```

Comment:
Calling this "loading screen" to emphasize it's meant for cases where the entire screen is empty. Not meant as a generic "spinner" component.